src/test/java/classes/EventTest.java

```diff
@@ -0,0 +1,85 @@
+package classes;
+
+import exceptions.IncorrectNoOfArgumentException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    Task tempTask = new Event("Birthday", "2023-02-05", "2023-02-05", "00:00", "23:59");
+    Task tempTask2 = new Event("Birthday", "2023-02-05", "2023-02-05", "", "23:59");
+    Task tempTask3 = new Event("Birthday", "2023-02-05", "2023-02-05", "00:00", "");
+    Task tempTask4 = new Event("Birthday", "2023-02-05", "2023-02-05", "", "");
+
+    @Test
+    public void getTaskInfoStatus() {
```

Comment:
Would recommend breaking this into several tests for readability.

Reply:
Could you perhaps clarify more on which part of the code that could be broken down into several tests?